### PR TITLE
✨Document-level infinite-scroll (#12945): Hide position fixed elements of nested pages.

### PR DESCRIPTION
**Pull request by emarchiori**
Uses a style attached in the shadow DOM to hide position fixed elements of nested pages.

Option of changing between positioned fixed elements of active page was considered, but decided against while only documents from one source are used.

@peterjosling

### CODE REVIEW

File: extensions/amp-document-recommendations/0.1/amp-document-recommendations.js [hunk 1]
```diff
@@ -176,6 +176,10 @@ export class AmpDocumentRecommendations extends AMP.BaseElement {
             if (this.nextArticle_ < this.config_.recommendations.length) {
               this.appendNextArticle_();
             }
+
+            installStylesForDoc(amp.ampdoc, CSS, () => {}, false, TAG);
```
[the remark this answers is not in the document]
Comment (emarchiori) [reply extensions/amp-document-recommendations/0.1/amp-document-recommendations.js]: Done.

**Comment by aghassemi** [comment]
@emarchiori LGTM, there is a conflict though, so I can't merge. please rebase and ping me. thanks!

**Comment by emarchiori** [comment]
@aghassemi Merge conflict resolved.

**Comment by emarchiori** [comment]
@aghassemi Should be fixed again.

**Comment by aghassemi** [comment]
@emarchiori merged

**Pull request merged**